components/Deck/ContentModulesPanel/DataSourcePanel/EditDataSource.js

```diff
@@ -94,7 +124,7 @@ class EditDataSource extends React.Component {
                     </div>
                     <div className="ui required field">
                         <label htmlFor="title">Title</label>
-                        <input type="text" ref="title" id="title" name="title" placeholder="Title" defaultValue={dataSource.title} autoFocus required />
```

Comment:
Is this change just cleanup? Seems the title remains a required field even after this.

Reply:
Yes, simple cleanup, to show the error with the semantic-ui validation handler.